src/app/components/Slideshow/sliderComponents/Slider.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import Slider from './Slider';
+import Images from './Slides';
+
+describe('<Slider />', () => {
+  it('should match its snapshot', () => {
+    const props = { slides: [], activeIndex: 0, isStopped: false };
+    const wrapper = shallow(<Slider {...props} />);
+
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should handle click slider indicator', () => {
+    const props = { slides: [] };
+    const wrapper = mount(<Slider {...props} />);
+
+    wrapper.setProps({ onClick: 'handleClickIndicator' });
```

Comment:
You should have checked if you are calling this method, but not changing it in props
```
it('should be able to call handleClickIndicator onClick', () => {
  const wrapper = shallow(<Slider {...props} />);
  const spy = jest.spyOn(wrapper.instance(), 'handleClickIndicator');

  wrapper.find('SliderIndicator').simulate('click');
  expect(spy).toHaveBeenCalled();
});
```